### PR TITLE
[BUGFIX] #24 Fix issue with news list view not rendering

### PR DESCRIPTION
**Pull request by sivaprasad1988**
Tasks:
* With the introduction of content elements with ext:news, need to tweak the TS a bit.

To-Do's:
* Needs to register the same for news_list_sticky,news_detail,news_selectedlist,news_datemenu,categorylist,news_searchform,news_searchresult,taglist

Resolves: ##24

### CODE REVIEW

**Comment by sivaprasad1988** [comment]
I changed the template and added the following snippet in my fluid template. In this way, I will receive all my content element-related information. 

```
"contentElementInfo":
  <f:format.raw>
      <f:format.json value="{
              0: contentObjectData
          }"/>
  </f:format.raw>,
```

Output https://www.screencast.com/t/lonwhzWoE

**Comment by tlueder** [comment]
Sure, but that removes the option to use the same configuration as for the other elements and one has to change the templates. Which is not ideal.

And also, why haven't you added this change to the templates here in your MR?
In this from your MR produces semi broken JSON.

**Comment by sivaprasad1988** [comment]
@tlueder Many thanks for the feedback. After having a closer look, I realize my mistake. Many thanks for the pointer. I corrected the TS a bit so that information on the content element, like appearance, etc., can be used with JSON results. 

JSON  Result https://www.screencast.com/t/CnG5emxav6HY

**Comment by juliasaurin** [comment]
@sivaprasad1988 Thanks for the Fix. It works now!!

File: Configuration/TypoScript/setup.typoscript [hunk 1]
```diff
@@ -29,3 +29,32 @@ tt_content.list {
...
+                    pluginName = Pi1
+                    controller = News
+                    view < plugin.tx_news.view
+                    persistence < plugin.tx_news.persistence
+                    settings < plugin.tx_news.settings
+                    settings {
+                      dateFormat = TEXT
+                      dateFormat = M d Y
+                      dateTimeFormat = TEXT
+                      dateTimeFormat = Y-m-d H:i:s
```
[the remark this answers is not in the document]
Comment (sivaprasad1988) [reply Configuration/TypoScript/setup.typoscript:54]: IMO, `dateFormat = TEXT` is irrelevant and it should be removed from this TS as this is a plugin setting. I tested this and works without `dateFormat = TEXT`.

File: Configuration/TypoScript/setup.typoscript [hunk 2]
```diff
@@ -29,3 +29,32 @@ tt_content.list {
...
+                    vendorName = GeorgRinger
+                    extensionName = News
+                    pluginName = Pi1
+                    controller = News
+                    view < plugin.tx_news.view
+                    persistence < plugin.tx_news.persistence
+                    settings < plugin.tx_news.settings
+                    settings {
+                      dateFormat = TEXT
+                      dateFormat = M d Y
```
[the remark this answers is not in the document]
Comment (sivaprasad1988) [reply Configuration/TypoScript/setup.typoscript:52]: The same goes here as well. `dateFormat = TEXT` should be removed.

**Pull request closed without merging**